ast/ArgParsing.cc

```diff
@@ -17,6 +17,7 @@ core::ParsedArg parseArg(const ast::ExpressionPtr &arg) {
             *cursor,
             [&](const ast::RestArg &rest) {
                 parsedArg.flags.isRepeated = true;
+                parsedArg.flags.isKwSplat = rest.isKwRest;
```

Comment:
Ideally I'd like to rename `isRepeated` to `isSplat` and also make it exclusive so the parameter could be either `isSplat` or `isKwSplat`. But it's an ambitious goal, because large chunks of argument dispatch depend on `isRepeated` being a combination of single and double splat. It would be to much of a lift to detangle it right now